index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
I'm wondering whether it'd be better to be strict about it, and only allow `gu-box` as an element. Here we're coupling the grid with another component.

I think it might be better to let the grid be simply a grid and nothing else. Inside the grid we place components, which we can then style. That way if we want to move the component we can do so easily. Just like line 70 above.